pkg/controller/jobframework/reconciler.go

```diff
@@ -236,7 +236,7 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 	ctx = ctrl.LoggerInto(ctx, log)
 
 	defer func() {
-		err = r.ignoreUnretryableError(log, err)
+		err = IgnoreUnretryableError(log, err)
```

Comment:
Is this change necessary, or just a temporary refactoring? If not necessary I prefer revert it, the diff is already large.

Reply:
Reverted.